compiler/modules/front.syntax/token.bal

```diff
@@ -569,6 +571,19 @@ public readonly class SourceFile {
     function scannedLine(int lineNumber) returns ScannedLine {
         return self.lines[lineNumber - 1];
     }
+
+    function validatePosition(Position pos) {
```

Comment:
This is not the right way to solve this problem.  Please do something more like what I suggested in the issue.

Reply:
fixed